contrib/neovide/update.py

```diff
@@ -0,0 +1,2 @@
+url = "https://github.com/neovide/neovide/tags"
+pattern = r'/archive/refs/tags/([\d.]+)(?=\.tar\.gz")'
```

Comment:
```suggestion
pattern = r'/archive/refs/tags/([\d.]+).tar.gz'
```

you don't need fancy captures

Reply:
sorry... idk anything about the python-specific regex stuff  
(skill issue basically)

will fix.

Reply:
wait shouldn't it be `\.tar\.gz` ?

Reply:
there's no realistic case where the . matters for being dot-only or any